go.mod

```diff
@@ -52,6 +50,7 @@ require (
 replace (
 	go.etcd.io/etcd/api/v3 => ./api
 	go.etcd.io/etcd/pkg/v3 => ./pkg
+	go.etcd.io/etcd/raft/v3 => ./raft
```

Comment:
This will work when building/testing the main module, but will not work for things that depend on etcd modules. What's the plan to make sure tagged versions of the main module refer to the correct versions of the submodules?

Reply:
replace are ignored [1] when some other module "foo" depends on etcd that depends on raft. 

foo will import  'go.etcd.io/etcd/server/v3  3.5.2'
In the repo go.mod for 'go.etcd.io/etcd/server/v3' will point on raft 'go.etcd.io/etcd/raft/v3  3.5.2', and the ./raft rule will get ignored. 

[1] https://github.com/golang/go/wiki/Modules#gomod

Reply:
We discussed this offline.
 
I will take care of updating automated release procedure such that it: 
    1. releases rc/pre version. tags it accordingly to https://golang.org/ref/mod#vcs-version, so each module gets its own tag. 
    2. runs the tests based on the cross-module dependencies and not the `replace` rules. 
    3. commits and tags final release.

Reply:
/cc @gyuho